mlir/lib/Analysis/Presburger/PresburgerRelation.cpp

```diff
@@ -1038,7 +1044,7 @@ PresburgerRelation PresburgerRelation::simplify() const {
 }
 
 bool PresburgerRelation::isFullDim() const {
-  return llvm::any_of(getAllDisjuncts(), [&](IntegerRelation disjunct) {
+  return llvm::any_of(getAllDisjuncts(), [](IntegerRelation disjunct) {
```

Comment:
can take disjunct by reference

Reply:
It can't take disjunct by const reference, because isFullDim isn't marked const, and cannot be marked const. Mutating the disjunct would change the behavior, and PresburgerRelation::isFullDim is marked const, so I wouldn't want to make this change as part of this patch.